app/controllers/trackings_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TrackingsController < ApplicationController
+  before_filter :authorize
+
+  def new
+    @tracking = Tracking.new
+    @clients = current_user.try(:clients)
+  end
+
+  def create
+    @tracking = Tracking.new(tracking_params)
+    if @tracking.save
+      redirect_to root_path
+      flash[:success] = "Work must go on!"
+    else
+      render 'new'
+    end
+  end
+
+  def update
+  end
+
+  def end
+    tracking = Tracking.find(params[:id])
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unscoped call to Tracking#find](https://www.codacy.com/app/conta-srdr/forsythia/file/2146965799/issues/source?bid=3139784&fileBranchId=3148684#l23)